### ISSUE TITLE
Button color for inactive windows on multiple screens

### ISSUE THREAD

**Issue by ishovkun** [psifidotos/applet-window-buttons#46]
Some themes (like e.g. SierraBreeze) vary the button color when the window is active/inactive.
This is very convenient for identifying the active window. The window-buttons applet does not support it, which inhibits the experience in multi-monitor setup.
In plain English: I have three screens and each has a maximized window. I'd like the applet to change the button colors for inactive windows so I can see which window has the focus. 

![Screenshot_20190822_124308](https://user-images.githubusercontent.com/11154303/63544616-74065f00-c4da-11e9-863e-ef464ff9eaa9.png)

**Comment by psifidotos** [comment]
Last time that was supported it broke the entire buttons experience because underlying plasma decorations produce errors

**Comment by ishovkun** [comment]
AWC hides the buttons when a screen with a maximized window doesn't have the focus. 
Can that be implemented instead?

**Comment by psifidotos** [comment]
Probably it might be a good idea to add in settings :

- draw inactive windows state
- consider windows to current screen only

and the user can choose what suits him best

**Comment by psifidotos** [comment]
Just checked that the filter by screen option exists already, have you enable it?

**Comment by ishovkun** [comment]
Yes, I have that enabled. I also got show when active window is maximized enabled. Therefore, when I switch to another screen, the buttons in the inactive screen with a maximized window are shown. That confuses me because I cannot say which screen has focus.

**Comment by psifidotos** [comment]
Plasma panel or Latte panel?

**Comment by psifidotos** [comment]
Please make also a screenshot of the applet settings to try to reproduce in my system. If you have screens filter enabled then if the active window is another screen then the window buttons should not be

**Comment by psifidotos** [comment]
shown. Sounds like a bug.

**Comment by ishovkun** [comment]
It's Latte. 
![Screenshot_20190822_173154](https://user-images.githubusercontent.com/11154303/63558696-cfe4de00-c502-11e9-8d4c-629cf93874a3.png)

**Comment by ishovkun** [comment]
In the screenshot I'm showing that all three window button applets are showing buttons even though 
there is only one window is active at a time. I'd like the buttons to be indicative of which screen I need to be looking at.

**Comment by psifidotos** [comment]
Show me a screenshot of Window buttons applet settings please

**Comment by psifidotos** [comment]
@ishovkun the issue for NOT hiding buttons in different screens is CONFIRMED and fixed through #47 

But I think you are right the DISABLED buttons is ALSO a good idea to add as an option the needs are the following:
- NEW visibility mode for showing buttons when at least one window is shown and not minimized in the current screen
- Paint Disabled state buttons when the last active window in the current screen is not active

**Comment by psifidotos** [comment]
@ishovkun feel free to report back if everything is ok now... I added two new options:

![image](https://user-images.githubusercontent.com/2216474/63581394-a3cb6a80-c59f-11e9-924f-6cc44bb91e2d.png)

**Comment by ishovkun** [comment]
Works like a charm. You rock!
